src/opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -0,0 +1,587 @@
+            return WV {
+                WV::NumWindows{ static_cast<std::size_t>(actDims[0]) },
+                WV::WindowSize{ static_cast<std::size_t>(actDims[6]) }
+            };
+        }
+
+       
+        
+        template <class IACNArray>
+        void staticContrib(const Opm::Action::ActionX& actx, IACNArray& iAcn)
+        {
+            //item [0 - 9]: are unknown, (=0)
+            
+            /*item [10] type of quantity for condition
+                1 for a field quantity (number of flowing producing wells)
+                2 for a well quantity 
+                3 for a (node) group quantity
+                //9 for a well group quantity
+                10 for DAY
+                11 for MNTH
+                12 for YEAR 
+            */
+            std::size_t ind = 0;
+            int noEPZacn = 26;
+            // write out the schedule Actionx conditions
+            const auto& actx_cond = actx.conditions();
+            for (auto cond_it = actx_cond.begin(); cond_it < actx_cond.end(); cond_it++) {
+                auto z_data = *cond_it;
+                // left hand quantity 
+                std::string lhsQtype = z_data.lhs.quantity.substr(0,1);
+                const auto it_lhsq = lhsQuantityToIndex.find(lhsQtype);
+                if (it_lhsq != lhsQuantityToIndex.end()) {
+                    iAcn[ind + 10] = it_lhsq->second;
+                }
+                else {
+                    std::cout << "Unknown condition type: " << z_data.lhs.quantity << std::endl;
+                    throw std::invalid_argument("Actionx: " + actx.name());
+                }
+                
+                /*item[11] - quantity type for rhs quantity
+                    1 - for field variables
+                    2 - for well variables?
+                    3 - for group variables 
+                    8 - for constant values
+                */
+                iAcn[ind + 11] = 8;
+                std::string rhsQtype = z_data.rhs.quantity.substr(0,1);
+                const auto it_rhsq = rhsQuantityToIndex.find(rhsQtype);
+                if (it_rhsq != rhsQuantityToIndex.end()) {
+                    iAcn[ind + 11] = it_rhsq->second;
+                }
+                
+                 /*item[12] - index for lhs type
+                    1 - for MNTH
+                    0 - for all other types
+                */
+                std::string lhsQ = z_data.lhs.quantity;
+                if ( lhsQ == "MNTH") {
+                    iAcn[ind + 12] = 1;
+                }
+                
+                /*item [13] - relates to operator
+                    OR   is 2
+                    AND is 1 
+                */
+                const auto it_logic_13 = logicalToIndex_13.find(z_data.logic);
+                if (it_logic_13 != logicalToIndex_13.end()) {
+                    iAcn[ind + 13] = it_logic_13->second;
+                }
+                else {
```

Comment:
It is OK to be defensive; but these symbols have been thoroughly checked/parsed already; at this point the `logicalToIndex_13` *really* should succeed. I would just `at`:
```
iAcn[ind + 13] = logicaltoIndex_13.at(z_data.logic);
```